$NetBSD: patch-aa,v 1.7 2010/03/19 13:22:42 obache Exp $

* add libxul-embedding to PACKAGED_IDS for xulrunner 1.9.2.
http://ruby-gnome2.svn.sourceforge.net/viewvc/ruby-gnome2?view=rev&revision=3752

* fix -Wl,-rpath syntax. [#2924766]
http://ruby-gnome2.svn.sourceforge.net/viewvc/ruby-gnome2?view=rev&revision=3727

* generate collected flags type. [#2925285]
http://ruby-gnome2.svn.sourceforge.net/viewvc/ruby-gnome2?view=rev&revision=3749

--- gtkmozembed/extconf.rb.orig	2009-09-23 00:43:01.000000000 +0000
+++ gtkmozembed/extconf.rb
@@ -4,6 +4,7 @@ extconf.rb for Ruby/GtkMozEmbed extentio
 
 PACKAGE_NAME = "gtkmozembed"
 PACKAGE_IDS = ["libxul-embedding-unstable",
+               "libxul-embedding",
                "xulrunner-gtkmozembed",
                "firefox-gtkmozembed",
                "seamonkey-gtkmozembed",
@@ -48,7 +49,7 @@ mozilla_path = mozilla_library_path.stri
 if mozilla_path
   # please comment the CFLAGS line if you don't want a default comp_path
   $CFLAGS << " -DDEFAULT_MOZILLA_FIVE_HOME='\"#{mozilla_path}\"' "
-  $LDFLAGS << " -Wl,-rpath #{mozilla_path}"
+  $LDFLAGS << " -Wl,-rpath,#{mozilla_path}"
 else
   $stderr.puts "${package_id}.pc cannot be found."
   exit 1
@@ -83,7 +84,8 @@ create_makefile_at_srcdir(PACKAGE_NAME, 
     result += [gtkmozembed_h] if File.exist?(gtkmozembed_h)
     result
   end
-  glib_mkenums(enum_type_prefix, headers, "GTK_TYPE_", ["gtkmozembed.h"])
+  glib_mkenums(enum_type_prefix, headers, "GTK_TYPE_", ["gtkmozembed.h"],
+		:force_flags => [/Flags/])
 end
 
 create_top_makefile
